lib/screens/app/profile/profile.dart

```diff
@@ -33,15 +33,19 @@ class _ProfileState extends State<Profile> {
 
   @override
   initState() {
+    var cachedProfile = ProfileNotifier.of(context).profile;
+    if (cachedProfile != null) _nameController.text = cachedProfile.nickname;
```

Comment:
In case we already made the call and switch tab in the app, it will re-display the name without latency and still fetch it from the network afterward. (Dumb caching 😃)